### PR TITLE
Cleanups from khmer onboarding

### PR DESCRIPTION
**Pull request by ctb**
* make 'add' a synonym for 'count' on hashtables;
* add logging into scripts in order to make '--quiet' work properly;
* add --quiet to args;

Unaddressed issues:

- [x] citation check had to be disabled. (See #1395)

Scripts updated:

- [x] filter-abund
- [x] filter-abund-single
- [x] abundance-dist
- [x] abundance-dist-single
- [x] load-into-counting
- [x] trim-low-abund

Could be used to address #878 too.

- [x] Is it mergeable?
- [x] `make test` Did it pass the tests?
- [x] `make clean diff-cover` If it introduces new functionality in
  `scripts/` is it tested?
- [x] `make format diff_pylint_report cppcheck doc pydocstyle` Is it well
  formatted?
- [x] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [x] Is it documented in the `ChangeLog`?
  http://en.wikipedia.org/wiki/Changelog#Format
- [x] Was a spellchecker run on the source code and documentation after
  changes were made?
- [x] Do the changes respect streaming IO? (Are they
  tested for streaming IO?)
- [x] Is the Copyright year up to date?

### CODE REVIEW

**Comment by standage** [comment]
#MakePythonGreatAgain

**Comment by ctb** [comment]
@standage a review would be welcome. Only thing I can't figure out how to do is deal with the citation test (in test_scripts.py, test_version) which checks to make sure that our 'info' command is properly spelling out the desired citations for each script, but I think that can be punted to an issue if this is ready for merge.

File: tests/test_filter_abund.py [hunk 1]
```diff
@@ -380,3 +380,65 @@ def test_outfile():
...
+
+    seqs = set([r.sequence for r in screed.open(outfile)])
+
+    assert len(seqs) == 1, seqs
+    assert 'GGTTGACGGGGCTCAGGG' in seqs
+
+    args = [n_counting_ht, n_infile]
+    utils.runscript(script, args, n_in_dir)
+
+    seqs = set([r.sequence for r in screed.open(n_infile)])
```
Comment (standage) [inline tests/test_filter_abund.py]: Did you mean to test the contents of `seqs`?

Comment (ctb) [inline tests/test_filter_abund.py]: Oh, I see, yeah, that's not useful. thx.

**Comment by standage** [comment]
Yeah, I was perplexed by the citation check when I was trying to resurrect Sherine's PR to promote a sandbox script. I punted on that one since it doesn't seem to be high priority, but I'm still trying to grok how all the argument parsers are built and how all the usages/versionstrings/vignettes are printed.

**Comment by ctb** [comment]
Ready for review and merge @standage

**Comment by standage** [comment]
LGTM.

**Pull request merged**